build.psm1

```diff
@@ -391,6 +391,13 @@ Fix steps:
         $Arguments += "/property:IsWindows=false"
     }
 
+    # Framework Dependent builds do not support ReadyToRun as it needs a specific runtime to optimize for.
+    # The property is set in Powershell.Common.props file.
+    # We override the property through the build command line.
+    if($Options.Runtime -like 'fxdependent*') {
+        $Arguments += "/property:PublishReadyToRun=false"
```

Comment:
does this override tho common properites?

Reply:
Command line properties has a priority over csjroj.

Reply:
Correct. Command line takes precedence.